### DISCUSSION TITLE
Wrong URL for submodule?

### DISCUSSION THREAD

**Discussion by mrusme** [oxen-io/lokinet#1922] [General]
I believe the URL to the following submodule is incorrect:

https://github.com/oxen-io/lokinet/blob/608dced827c92645ce74478378985400cb104a6f/.gitmodules#L22

**Comment by majestrate** [comment]
trailing `/` ?

**Comment by majestrate** [comment]
i dont think we even use that submodule at all, removed in 0d0295e2d

**Reply by mrusme** [reply]
Thanks, that was quick! Yes, now cloning works. :-)
